pkgs/development/python-modules/woob/default.nix

```diff
@@ -55,17 +51,18 @@ buildPythonPackage rec {
     pyyaml
     requests
     rich
-    termcolor
```

Comment:
why removing these dependencies?

Reply:
Because we always miss things in the python-updates bulk update.

Reply:
I searched the upstream code for the dependency and couldn't find it.